src/types/loans.ts

```diff
@@ -37,11 +37,11 @@ interface LoanAsset {
 
 // Enables easier access to data by asset ticker key.
 type TickerToData<T> = {
-    [ticker: string]: T
-}
+    [ticker: string]: T;
+};
 
 interface LoanMarket {
-    lendTokenId: number
+    lendTokenId: number;
 }
 
-export type { LoanPosition, LendPosition, BorrowPosition, LoanAsset, TickerToData, LoanMarket };
+export type { LoanPosition, LendPosition, BorrowPosition, LoanAsset, TickerToData, LoanMarket, LoanReward };
```

Comment:
If you're using `LoanReward` for both lending and borrowing, a better naming for this type would be `SubsidyReward`

Reply:
Nice suggestion, ta